media/js/wiki.js

```diff
@@ -847,5 +847,90 @@
     if($('details').length){
         initDetailsTags();
     }
+    
+    function openJSFiddle(title, htmlCode, cssCode, jsCode) {
+       var $form = $('<form method="post" target="_blank" action="https://jsfiddle.net/api/mdn/">'
+            + '<input type="hidden" name="html" />'
+            + '<input type="hidden" name="css" />'
+            + '<input type="hidden" name="js" />'
+            + '<input type="hidden" name="title" />'
+            + '<input type="hidden" name="wrap" value="b" />'
+            + '<input type="submit" />'
+        + '</form>');
+       $form.find('input[name=html]').val(htmlCode);
+       $form.find('input[name=css]').val(cssCode);
+       $form.find('input[name=js]').val(jsCode);
+       $form.find('input[name=title]').val(title);
+       $form.submit();
+    }
+    
+    function openCodepen(title, htmlCode, cssCode, jsCode) {
+       var $form = $('<form method="post" target="_blank" action="http://codepen.io/pen/define">'
```

Comment:
I think creating elements programatically will make the code too long and unreadable. But I'm happy to do it, if this breaks our style guide.

Also, should I move my code into separate file. IIRC, we want to waffle flag this feature as well as A/B test.  Is this part as simple as creating separate js file, or do I need to do waffle flag checking in js also?

Let me know if there is anything else that needs to be fixed. I want this feature in production asap :)